.gitignore

```diff
@@ -29,7 +30,7 @@ src/ontology/reports/*
 !src/ontology/reports/README.md
 !src/ontology/reports/*excluded_terms_in_mondo_xrefs.tsv
 !src/ontology/reports/*exclusion_reasons.robot.template.tsv
-!src/ontology/reports/*term_exclusions.txt
+!src/ontology/reports/*excluded_terms.txt
```

Comment:
@sabrinatoro @nicolevasilevsky FYI, I just did a rename of `reports/term_exclusions.txt` to `reports/excluded_terms.txt`. This is for alphabetical consistency w/ the other aggregated reports: `exclusion_reasons.robot.template.tsv`, `excluded_terms_in_mondo_xrefs.tsv`. I updated the `reports/README.md`, etc, to match the name change.

@matentzn FYI just adding this small change to the slurp PR, but I can make this commit directly to `main` or do a separate PR if you prefer.